src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json

```diff
@@ -1,480 +1,480 @@
+    "orderItemColumnPaymentMethod": "Zahlungsart",
+    "orderItemColumnShippingMethod": "Versandart",
+    "orderItemColumnActions": "Aktionen",
+    "orderItemColumnName": "Produkt",
+    "orderItemColumnQuantity": "Anzahl",
+    "orderItemColumnPrice": "Stückpreis",
+    "orderItemColumnTotal": "Summe",
+    "orderItemDate": "Von:",
+    "orderItemNumber": "Bestellnummer:",
+    "orderItemPaymentMethod": "Zahlungsart:",
+    "orderItemShippingMethod": "Versandart:",
+    "orderItemTracking": "Paketverfolgung:",
+    "orderItemShippingcosts": "Versandkosten:",
+    "orderItemNetTotal": "Gesamtsumme (netto):",
+    "orderItemTotal": "Gesamtsumme:",
+    "orderItemComment": "Unser Kommentar",
+    "orderItemCustomerComment": "Ihr Kommentar",
+    "orderItemInfoContent": "Inhalt:",
+    "orderItemInfoCurrentPrice": "Aktueller Einzelpreis",
+    "orderItemInfoFree": "GRATIS",
+    "orderItemInfoNotAvailable": "Das Produkt ist für eine Folgebestellung nicht mehr verfügbar",
+    "orderActionView": "Anzeigen",
+    "orderActionHide": "Ausblenden",
+    "orderInfoNoDispatch": "Nicht angegeben",
+    "orderLinkRepeat": "Bestellung wiederholen",
+    "newsletterTitle": "Newsletter-Einstellungen",
+    "newsletterSettings": "Ja, ich möchte regelmäßig den kostenlosen Newsletter von %shopname% erhalten! Ich weiß, dass ich mich hier jederzeit wieder von dem Newsletter abmelden kann.",
+    "passwordResetTitle": "Haben Sie ihr Passwort vergessen?",
+    "passwordResetInfo": "Wenn Sie das Passwort für Ihr Konto vergessen haben, können Sie hier ein neues definieren. Wenn Sie das neue Passwort speichern, wird Ihr altes Passwort ungültig.",
+    "passwordOptInInfo": "Wir senden Ihnen eine Bestätigungs-E-Mail. Klicken Sie auf den darin enthaltenen Link, um Ihr Passwort zu ändern."
+  },
+  "address": {
+    "streetLabel": "Straße und Hausnummer",
+    "streetPlaceholder": "Straße und Hausnummer eingeben...",
+    "additionalField1Label": "Adresszusatz 1",
+    "additionalField1Placeholder": "Adresszusatz eingeben ...",
+    "additionalField2Label": "Adresszusatz 2",
+    "additionalField2Placeholder": "Adresszusatz eingeben ...",
+    "zipcodeLabel": "PLZ",
+    "zipcodePlaceholder": "Postleitzahl eingeben ...",
+    "cityLabel": "Ort",
+    "cityPlaceholder": "Ort eingeben ...",
+    "countryLabel": "Land",
+    "countryPlaceholder": "Land auswählen ...",
+    "countryStateLabel": "Bundesland",
+    "countryStatePlaceholder": "Bundesland auswählen ...",
+    "phoneNumberLabel": "Telefonnummer",
+    "phoneNumberPlaceholder": "Telefonnummer eingeben ...",
+    "companyNameLabel": "Firma",
+    "companyNamePlaceholder": "Firma eingeben ...",
+    "companyDepartmentLabel": "Abteilung",
+    "companyDepartmentPlaceholder": "Abteilung eingeben ...",
+    "companyVatLabel": "Umsatzsteuer-ID",
+    "companyVatPlaceholder": "Umsatzsteuer-ID"
+  },
+  "checkout": {
+    "itemCounter": "{1} 1 Produkt|]1,Inf[ %count% Produkte",
```

Comment:
Isn't this the snippet of a word counter in some WYSIWYG-Editor? I think @GitEvil told me once that it's as it should be.

Reply:
pls check source: https://github.com/shopware/platform/blob/3defa231aab16b3682783ba31d1140e331389cbb/src/Storefront/Resources/snippet/en_GB/storefront.en-GB.json#L264

Reply:
It's in a different place, than I thought it would be. :+1: (That would be a case, where I'd trusted the germen version more :P)